sql/step_a113_alter_table_medicalinventory.sql

```diff
@@ -0,0 +1,12 @@
+-- Step 1: Drop the foreign key constraint
+ALTER TABLE OH_MEDICALDSRINVENTORY 
+DROP CONSTRAINT oh_medicaldsrinventory_ibfk_2;
+
+-- Step 2: Change the data type of the foreign key column
+ALTER TABLE OH_MEDICALDSRINVENTORY 
+MODIFY MINVT_WRD_ID_A CHAR(3);
```

Comment:
Although optional I would specify the object for better readability

```suggestion
ALTER TABLE OH_MEDICALDSRINVENTORY 
MODIFY COLUMN MINVT_WRD_ID_A CHAR(3);
```